main.py

```diff
@@ -22,6 +22,7 @@
 
 
 app = Flask(__name__)
+app.config.from_object('config.DevConfig')
```

Comment:
How can different configs be loaded depending on the environment? E.g. load the dev config locally, but the prod config when running in production.

Here's a suggestion that I found after some Googling: https://stackoverflow.com/questions/25745390/how-to-serve-different-config-settings-in-flask-app-to-uwsgi-using-create-app

Or, feel free to choose some other method. I think that the requirement is that different configs can be loaded in different environments.

Reply:
I see. How should I load configs in the test file then? Should I still do it directly as `main.app.config.from_object('config.DevConfig')`, or should test cases follow the same environment that the app is using?

Reply:
I think it's still okay to load directly in the unit tests. This was more-so a practical concern. E.g how can we set the config when we deploy to prod?

Reply:
Also in general, is it ok to set environment variables in a .env file and loading using the dotenv library? Since we're using a Docker container I felt that it makes sense.

Reply:
Yes, I assume we can just set the correct environment variables in the Dockerfile.